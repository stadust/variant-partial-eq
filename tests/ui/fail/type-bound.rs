use std::marker::PhantomData;
use variant_partial_eq::VariantPartialEq;

#[derive(VariantPartialEq)]
struct MyStruct<T: PartialEq> {
    ghost: PhantomData<T>
}

fn main() {}