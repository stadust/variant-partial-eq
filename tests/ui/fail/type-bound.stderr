error: #[derive(VariantPartialEq)] does not support type bounds
 --> tests/ui/fail/type-bound.rs:5:20
  |
5 | struct MyStruct<T: PartialEq> {
  |                    ^^^^^^^^^
